src/ViewLayer/RaceModeDisplay/RaceModeDashboardView.cpp

```diff
@@ -71,6 +72,18 @@ void RaceModeDashboardView::connectBattery(BatteryPresenter& batteryPresenter)
             this, SLOT(prechargeStateReceived(QString)));
     connect(&batteryPresenter, SIGNAL(packNetPowerReceived(double)),
             this, SLOT(packNetPowerReceived(double)));
+    connect(&batteryPresenter, SIGNAL(auxVoltageReceived(int)),
```

Comment:
Battery presenter no longer has the aux voltage or precharge state, that's all in the Aux Presenter now!